src/main/resources/db/migration/V1_create_movies.sql

```diff
@@ -0,0 +1,9 @@
+DROP TABLE IF EXISTS movie CASCADE;
+CREATE TABLE movie
+(
+    id         INTEGER GENERATED BY DEFAULT AS IDENTITY,
+    legacy_id  INTEGER,
```

Comment:
É importante alterar a coluna LEGACY_ID para manter o padrão e também o datatype. Segue abaixo o exemplo:

"ID_LEGACY NUMBER (24, 0) NOT NULL,"